apps/Gruntfile.js

```diff
@@ -331,27 +331,64 @@ module.exports = function (grunt) {
     allFilesDest.push(outputDir + app + '.js');
   });
 
-  // Use command-line tools to run browserify (faster/more stable this way)
-  var browserifyExec = 'mkdir -p build/browserified &&' +
-      (envOptions.dev ? '' : ' NODE_ENV=production') + // Necessary for production Redux
-      ' `npm bin`/browserifyinc' +
-      ' -g [ browserify-global-shim ]' +
-      ' --cachefile ' + outputDir + 'browserifyinc-cache.json' +
-      ' -t [ babelify --compact=false --sourceMap --sourceMapRelative="$PWD" ]' +
+  /**
+   * Generates a command line string to execute browserify with the specified options.
+   * Available options are:
+   *
+   *   {bool} globalShim - boolean for whether or not to turn on
+   *                       the browserify-global-shim transform
+   *   {string} cacheFile - file name where the browserify cache should be stored
+   *   {string[]} srcFiles - list of src files to pass to browserify
+   *   {string[]} destFiles - list of destination file paths to pass to browserify
+   *   {bool} factorBundle - whether or not to use the factor-bundle plugin to
+   *                         extract common code into a separate file.
```

Comment:
Nice documentation.  If I can be super-picky, we try to use [JSDoc](http://usejsdoc.org/) which would change this comment to something like this:

```
/**
 * Generates a command line string to execute browserify with the specified options.
 * 
 * @param {!Object} options
 * @param {boolean} [options.globalShim] - whether or not to turn on the
          browserify-global-shim transform
 * @param {!string} options.cacheFile - file name where the browserify cache should be stored
 * @param {!string[]} options.srcFiles - list of src files to pass to browserify
 * @param {!string[]} options.destFiles - list of destination files to pass to browserify
 * @param {boolean} [options.factorBundle] - whether or not to use the factor-bundle plugin
          to extract common code into a separate file.
 */
```

Of course, we're not generating that JSDoc anywhere, so I'm not sure how much it matters :)